### ISSUE TITLE
An unexpected error has occurred with MSMBuilder (version 3.3.0) Command Line Tutorial

### ISSUE THREAD

**Issue by lguarneros** [msmbuilder/msmbuilder#657]
Hi everybody!
I have the next error after execute the commands on Tutorial Command Line 

```
$msmb AtomPairsFeaturizer --out pair_features --pair_indices AtomIndices.txt --top alanine_dipeptide/ala2.pdb --trjs "alanine_dipeptide/*.dcd"
usage: msmb AtomPairsFeaturizer [-h] [--chunk CHUNK] [-o OUT]
                                [--stride STRIDE] [--top TOP] --transformed
                                TRANSFORMED --trjs TRJS --pair_indices
                                PAIR_INDICES [--periodic]
                                [--exponent EXPONENT]
msmb AtomPairsFeaturizer: error: argument --transformed is required
```

I give the option --transformed and run ok.

The next command of Tutorial Command Line have a error:
```
$msmb tICA -i pair_features --out tica_model --transformed tica_trajs --n_components 4
```
Send this traceback:
```
$msmb tICA -i pair_features --out tica_model --transformed tica_trajs --n_components 4
tICA(gamma=0.05, lag_time=1, n_components=4, weighted_transform=False)
Fitting model...
*********
*RESULTS*
*********
An unexpected error has occurred with MSMBuilder (version 3.3.0), please
consider sending the following traceback to MSMBuilder GitHub issue tracker at:
            https://github.com/msmbuilder/msmbuilder/issues

Traceback (most recent call last):
  File "/opt/anaconda/bin/msmb", line 6, in <module>
    sys.exit(main())
  File "/opt/anaconda/lib/python2.7/site-packages/msmbuilder-3.3.0-py2.7-linux-x86_64.egg/msmbuilder/scripts/msmb.py", line 28, in main
    app.start()
  File "/opt/anaconda/lib/python2.7/site-packages/msmbuilder-3.3.0-py2.7-linux-x86_64.egg/msmbuilder/cmdline.py", line 440, in start
    instance.start()
  File "/opt/anaconda/lib/python2.7/site-packages/msmbuilder-3.3.0-py2.7-linux-x86_64.egg/msmbuilder/commands/fit_transform.py", line 61, in start
    print(self.instance.summarize())
  File "/opt/anaconda/lib/python2.7/site-packages/msmbuilder-3.3.0-py2.7-linux-x86_64.egg/msmbuilder/decomposition/tica.py", line 446, in summarize
    timescales=self.timescales_[:5], eigenvalues=self.eigenvalues_[:5])
  File "/opt/anaconda/lib/python2.7/site-packages/msmbuilder-3.3.0-py2.7-linux-x86_64.egg/msmbuilder/decomposition/tica.py", line 208, in timescales_
    self._solve()
  File "/opt/anaconda/lib/python2.7/site-packages/msmbuilder-3.3.0-py2.7-linux-x86_64.egg/msmbuilder/decomposition/tica.py", line 168, in _solve
    if not np.allclose(self.offset_correlation_, self.offset_correlation_.T):
  File "/opt/anaconda/lib/python2.7/site-packages/msmbuilder-3.3.0-py2.7-linux-x86_64.egg/msmbuilder/decomposition/tica.py", line 223, in offset_correlation_
    two_N = 2 * (self.n_observations_ - self.lag_time * self.n_sequences_)
TypeError: unsupported operand type(s) for *: 'int' and 'NoneType'
```

Can help me please?

**Comment by mpharrigan** [comment]
cc #651 

When you add the `--transformed` flag, what do you set it to

**Comment by mpharrigan** [comment]
This is fixed in #658. Simply change `--out` to `--transformed`. Note that `--out` was deprecated in 3.2, but we missed updating the tutorial
